processor/src/main/java/org/mapstruct/ap/internal/model/BeanMappingMethod.java

```diff
@@ -95,13 +95,10 @@ public class BeanMappingMethod extends NormalTypeMappingMethod {
     private final Type returnTypeToConstruct;
     private final BuilderType returnTypeBuilder;
     private final MethodReference finalizerMethod;
-
```

Comment:
Can you please revert all your changes in this file since they are only removing and adding lines

Reply:
Thank you for pointing it out, it has been changed